metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_audit.py

```diff
@@ -104,21 +116,29 @@ def raw_table_name(self):
         return f"{self.project_id}.{self.dataset}.{self.table}"
 
     def get_table_display_name(self) -> str:
+        """
+        Returns table display name to be used for DataHub
+            - removes shard suffix (table_yyyymmdd-> table)
+            - removes wildcard part (table_yyyy* -> table)
+            - remove time decorator (table@1624046611000 -> table)
+        """
         shortened_table_name = self.table
-        # if table name ends in _* or * then we strip it as that represents a query on a sharded table
+        # if table name ends in _* or * or _yyyy* or _yyyymm* then we strip it as that represents a query on a sharded table
```

Comment:
Seems like we support any series of numbers between the `_` and `*` in the regex, so you could technically do something like `_yy*` or `_yyyym*` if you wanted. Not sure the best way to reflect this in a comment though